conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
We don't need this environment. There are no new globals, and we definitely don't want to add a new environment for each new edition.